packages/kit/src/views/Home/components/NFTListView/index.tsx

```
@@ -156,6 +163,9 @@ function NFTListView(props: IProps) {
       renderItem={handleRenderItem}
       ListHeaderComponent={<NFTListHeader filteredNfts={filteredNfts} />}
       ListEmptyComponent={searchKey ? <EmptySearch /> : <EmptyNFT />}
+      ListFooterComponent={
+        <>{addPaddingOnListFooter ? <Stack h="$16" /> : null}</>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)